### ISSUE TITLE
show what key failed on 403

### ISSUE THREAD

**Issue by grosser** [hashicorp/vault-ruby#120]
permission errors can happen on a per key basis, so would be nice to see what failed ...

```
  Vault::HTTPClientError: The Vault server at `https://staging.hvault.service.consul:8200' responded with a 403.
  Any additional information the server supplied is shown below:
    * permission denied
  Please refer to the documentation for help.
```

**Comment by evanphx** [comment]
This was addressed via #33

**Comment by evanphx** [comment]
This is likely an enhancement that vault should implement rather than this gem. The caller of the gem can at least wrap the read call to request this exception to see what was being read.
